### PR TITLE
Template

### PR DESCRIPTION
**Pull request by Megha-Bose**
Latest template

### CODE REVIEW

**Comment by Pk13055** [comment]
Fixes #6

**Comment by Pk13055** [comment]
Also, you’ll have to re submit this, there shouldn’t be a `merge from OSDG/IIIT-H` in the commits, you have pass the `—rebase` flag when pulling

**Comment by Megha-Bose** [comment]
Is it fine now?

File: frontend/src/plugins/vuetify.js [hunk 1]
```diff
@@ -4,6 +4,16 @@ import 'vuetify/dist/vuetify.min.css'
...
 Vue.use(Vuetify)
 
-const opts = {}
+const opts = {
+    theme: {
+        themes: {
+          light: {
+            primary: '#3f51b5',
+            secondary: '#ffe6ff',
+            anchor: '#8c9eff',
```
Comment (Pk13055) [inline frontend/src/plugins/vuetify.js:13]: So is this a default vuetify configuration method? Like if we add a `dark: { ... }` would we be able to toggle between those modes?

**Pull request merged**